### ISSUE TITLE
bad idea

### ISSUE THREAD

**Issue by SergSlipushenko** [Iaroslavrul/Poligon_finder#6]
https://github.com/Iaroslavrul/Poligon_finder/blob/ad99798590ff7bb0d9c7ba1ce61d82c87b4e243a/geo_test.py#L63

**Comment by Iaroslavrul** [comment]
You are right. But I was forced to use this trick, because the resulting area did not differ by more than 1 cm2, but this was enough to ensure that the equality condition is not fulfilled.
7651213.867215546 
7651213.867214167
To be on the safe side accuracy to two decimal places.
Thank you!
